uk.co.studiopond.spiderbro2.yml

```diff
@@ -0,0 +1,31 @@
+app-id: uk.co.studiopond.spiderbro2
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: spiderbro2_linux
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+modules:
+  - shared-modules/glu/glu-9.json
+  - name: Spiderbro-2
+    buildsystem: simple
+    build-commands:
+      - unzip -o Spiderbro2-Flatpak.zip
+      - install -Dm755 spiderbro2_linux /app/bin/spiderbro2_linux
+      - install -Dm755 polybius2_linux /app/bin/polybius2_linux
+      - install -Dm755 polybius_linux /app/bin/polybius_linux
+      - install -Dm755 vameforspiderbroforspiderbro2_linux /app/bin/vameforspiderbroforspiderbro2_linux
+      - cp -r data /app/bin/data
+      - unzip -o polydata2.zip -d /app/bin
+      - unzip -o polydata.zip -d /app/bin
+      - unzip -o spiderbrodata.zip -d /app/bin
+      - install -D uk.co.studiopond.spiderbro2.svg /app/share/icons/hicolor/scalable/apps/uk.co.studiopond.spiderbro2.svg
+      - install -D uk.co.studiopond.spiderbro2.desktop /app/share/applications/uk.co.studiopond.spiderbro2.desktop
+      - install -D uk.co.studiopond.spiderbro2.metainfo.xml /app/share/metainfo/uk.co.studiopond.spiderbro2.metainfo.xml
+    sources:
+     - type: file
```

Comment:
Use `archive` type and drop the extraction command, it will get extracted automatically

https://docs.flatpak.org/en/latest/module-sources.html#archive-sources

Reply:
So I made the change you requested, but upon testing I could not get it to work. I tried adding dest-filename, archive-type and I played around with strip-components but to no avail. From what I can tell, it doesn't seem to be downloading or extracting and I'm not sure why.

Here is the output of the build command if that helps.
https://pastebin.com/izN2nh26

Reply:
Add the suggested changes and it should work